lib/client.ts

```diff
@@ -0,0 +1,42 @@
+import { Kafka } from 'kafkajs';
+
+export const produceAndConsumeMessage = async function (
+  broker: string,
+  credentials: { clientID: string; clientSecret: string },
+  topic: string
+) {
+  const kafka = new Kafka({
+    brokers: [broker],
+    // authenticationTimeout: 10000,
+    // reauthenticationThreshold: 10000,
+    ssl: true,
+    sasl: {
+      mechanism: 'plain', // scram-sha-256 or scram-sha-512
+      username: credentials.clientID,
+      password: credentials.clientSecret
+    }
+  });
+
+  const producer = kafka.producer();
+
+  await producer.connect();
+  await producer.send({
```

Comment:
You should check the result or? For example make sure that messages was send without error

Reply:
@Frawless the action of producing and consuming messages on this test as far as I know is just a mean to get to the final goal of finally check the consumer shows up on the table inside 'Consumer Groups' Tab

Reply:
well in that case you don't need producer? Anyway you should check producer errors. I have it in my branch atm, it is not perfect but it is more robust.

Reply:
I might be wrong but if you start the producer before the consumer, send a message and disconnect the producer, create the consumer and listen for messages, is the consumer ever really gonna receive the message? 
As @Frawless mentioned, to create a consumer group, the producer might not be needed.

Reply:
It doesn't need the producer to be working once it has left the message for the consumer to read it @mmusil

Reply:
Can I resolve this conversation? @Frawless @mmusil